src/js/components/user-management/userform.js

```diff
@@ -1,49 +1,133 @@
 import React from 'react';
+import {
+  Button,
+  Checkbox,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  FormControl,
+  ListItemText,
+  InputLabel,
+  MenuItem,
+  Select
+} from '@material-ui/core';
 
 import Form from '../common/forms/form';
 import TextInput from '../common/forms/textinput';
 import PasswordInput from '../common/forms/passwordinput';
-import FormButton from '../common/forms/formbutton';
```

Comment:
this commit also changes the buttons, but commit message says otherwise.

Reply:
this specific commit just removes the reference to the button, as in the same step the prior usage of the button is removed since the entire dialog was refactored into its own component (where the `FormButton` is replaced with the standard `Button` component)